### PR TITLE
Adding information about how to add a hidden folder

### PR DESCRIPTION
**Pull request by matthiaz**

### CODE REVIEW

File: src/configuration/app/storage.md [hunk 1]
```diff
@@ -114,6 +114,17 @@ web:
 
 See the [web locations](/configuration/app/web.md) documentation for more details.
 
+## Unable to mount hidden folder?
+
+We ignore every YAML key that starts with a dot. This causes a mount like '.myhiddenfolder' to be ignored. If you want to mount a hidden folder, you'll have to prepend it with a `/`
```
Comment (chadwcarlson) [inline src/configuration/app/storage.md]:
```suggestion
Platform.sh ignores YAML keys that start with a dot. This causes a mount like `.myhiddenfolder` to be ignored. If you want to mount a hidden folder, you'll have to prepend it with a `/`:
```

File: src/configuration/app/storage.md [hunk 2]
```diff
@@ -114,6 +114,17 @@ web:
...
+
+```yaml
+mounts:
+  '/.myhiddenfolder':
+    source: local
+    source_path: '.myhiddenfolder'
```
Comment (chadwcarlson) [inline src/configuration/app/storage.md]:
```suggestion
    source_path: 'myhiddenfolder'
```

Comment (chadwcarlson) [reply src/configuration/app/storage.md]: This one's your call. In our templates we don't keep mount source paths hidden to keep it simple and visible, so this suggestion is more of keeping it consistent with that choice.

Comment (Crell) [reply src/configuration/app/storage.md]: Dot files are annoying, so let's not use them in source paths.

File: src/configuration/app/storage.md [hunk 3]
```diff
@@ -114,6 +114,17 @@ web:
 
 See the [web locations](/configuration/app/web.md) documentation for more details.
 
+## Unable to mount hidden folder?
```
Comment (Crell) [inline src/configuration/app/storage.md]:
```suggestion
## Why can't I mount a hidden folder?
```

**Pull request merged**